data/movie/Hancock 4K (2008)/hancock/disc02-summary.txt

```diff
@@ -0,0 +1,295 @@
+Name: Hancock
+Source file name: 00530.mpls
+Duration: 0:01:55
+Type: Extras
```

Comment:
This should be `Type: Extra` instead of `Type: Extras` Looks like you need to change this throughout this file

Reply:
Sure, I apologise, I was trying to get the basic information in place and got lazy with copy/paste :P ill sort it.